google/cloud/bigtable/testing/table_integration_test.cc

```diff
@@ -77,18 +77,20 @@ std::string TableTestEnvironment::RandomTableId() {
   // This value was discovered by trial and error, it is not documented in the
   // proto files.
   constexpr int kMaxTableIdLength = 50;
-  static std::string const prefix = "table-";
-  return CreateRandomId(prefix,
-                        static_cast<int>(kMaxTableIdLength - prefix.length()));
+  static char const prefix[] = "table-";
+  static_assert(kMaxTableIdLength > sizeof(prefix), "prefix is too long");
+  constexpr int sample_count = kMaxTableIdLength - sizeof(prefix) + 1;
```

Comment:
Do you not get the same `size_t` to `int` warning on these, or do we avoid it now that it is at `constexpr` time?

Reply:
I was checking that (on two platforms, thus the delay) before saying PTAL, but yes, it works.

Reply:
Cool.  Thanks.